creepy/subprocess/instance.py

```diff
@@ -0,0 +1,44 @@
+import typing
+from functools import partial
+from inspect import Signature, Parameter
+
+
+if typing.TYPE_CHECKING:
+    from .pypen import Pypen
+
+
+def instantiate(process: "Pypen"):
```

Comment:
Maybe to make it a Pypen's method? Also, as I understand the word "instantiate" means creating instance of some class. For example, when you do `Pypen(…)` you instantiate object of Pypen class.

Reply:
By the way, the another option is to merge proxy object with Pypen.

Reply:
What about `wait` and `request` methods of `Pypen`?

Reply:
I think I overestimated how often `Pypen` is used in another places. It is possible to make it like you propose, will do it then!